inputs/radiation/athinput.rad_linearwave

```diff
@@ -0,0 +1,81 @@
+<comment>
+problem   = Radiation beam test
+reference = 
+configure = --prob=radwave
+
+<job>
+problem_id = radwave  # problem ID: basename of output filenames
+
+<output1>
+file_type = hst   # History data dump
+dt        = 0.01  # time increment between outputs
+
+#<output2>
```

Comment:
remove commented-out lines, or uncomment them

Reply:
I have fixed all the issues with comments that I can find. Let me know if anything is missed.